azurerm/internal/services/healthcare/healthcare_service_resource.go

```diff
@@ -188,6 +188,16 @@ func resourceHealthcareService() *schema.Resource {
 				},
 			},
 
+			"public_network_access": {
```

Comment:
could we change this to
```suggestion
			"public_network_access_enabled": {
```
and make it a boolean to be consistent with the rest of these propeties?

Reply:
Sure, will address tomorrow my morning.